src/Files.App/ViewModels/Properties/Items/FileProperties.cs

```diff
@@ -97,8 +97,11 @@ public override async Task GetSpecialPropertiesAsync()
 
 			ViewModel.ItemSizeVisibility = true;
 			ViewModel.ItemSize = Item.FileSizeBytes.ToLongSizeString();
-			ViewModel.ItemSizeOnDisk = NativeFileOperationsHelper.GetFileSizeOnDisk(Item.ItemPath)?.ToLongSizeString() ??
-				string.Empty;
+
+			// Only load the size for items on the device
+			if (Item.SyncStatusUI.SyncStatus is not CloudDriveSyncStatus.FileOnline and not CloudDriveSyncStatus.FolderOnline)
```

Comment:
This is also not necessary in my testing

Reply:
Without it, the files download on my device.

Reply:
All type of files? For me they show this, which is correct for online files
<img width="449" alt="image" src="https://github.com/files-community/Files/assets/9673091/d03a2867-7e71-4c2e-9885-9bb7fc70d18b">

Reply:
I'll test again, but previously it would download any file type when calculating the size on disk.

Reply:
Yeah, it keeps downloading if I test with a txt file.

Reply:
I don't think there is any downside to keeping this check, but it would be nice if we could do something better for `OfflinePartial` folders.